src/views/StandaloneDirectorsFiling.vue

```diff
@@ -98,7 +98,8 @@
                     :isCertified.sync="isCertified"
                     :certifiedBy.sync="certifiedBy"
                     :entityDisplay="displayName()"
-                    :message="certifyText(FilingCodes.DIRECTOR_CHANGE_OT)"
+                    :message="certifyText(isBComp ? FilingCodes.DIRECTOR_CHANGE_BC
```

Comment:
This ternary expression is used multiple times in this module. I recommend making it a getter (so it can be cached, and for cleaner code throughout).

Reply:
Absolutely, will do!

Reply:
Updated in next commit 👍